prowler/providers/kubernetes/services/etcd/etcd_client_cert_auth/etcd_client_cert_auth.py

```diff
@@ -0,0 +1,21 @@
+from prowler.lib.check.models import Check, Check_Report_Kubernetes
+from prowler.providers.kubernetes.services.etcd.etcd_client import etcd_client
+
+
+class etcd_client_cert_auth(Check):
+    def execute(self) -> Check_Report_Kubernetes:
+        findings = []
+        for pod in etcd_client.etcd_pods:
+            report = Check_Report_Kubernetes(self.metadata())
+            report.namespace = pod.namespace
+            report.resource_name = pod.name
+            report.resource_id = pod.uid
+            report.status = "FAIL"
+            report.status_extended = f"Etcd does not have client certificate authentication enabled in pod {pod.name}."
+            for container in pod.containers.values():
```

Comment:
You should include a break for the loop, set `PASS` if all are set to pass. You can use a variable to check that.